cli/fossilize_replay_windows.hpp

```diff
@@ -640,14 +640,31 @@ static int run_master_process(const VulkanDevice::Options &opts,
 	vector<ProcessProgress> child_processes(processes);
 	vector<HANDLE> wait_handles;
 
+	size_t requested_graphic_pipelines = replayer_opts.end_graphics_index - replayer_opts.start_graphics_index;
```

Comment:
Is this range checked? Seems like it can easily go to UINT32_MAX without being clamped against real size of archive.